src/BackendPkg/NewRouter.go

```diff
@@ -280,6 +276,7 @@ func handleLogout(w http.ResponseWriter, r *http.Request, sessionCookie *string)
 		pantryInterface = interfaceRefresh
 		dealsInterface = interfaceRefresh
 		recipesInterface = interfaceRefresh
+		listInterface = interfaceRefresh
```

Comment:
just forgot to add this when we did E2E list implementation

Reply:
So it was this loop here that was giving us issues?